src/pages/Survey/Question/SingleQuestion.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function SingleQuestion({
+  surveyLists,
+  handleInputValue,
+  singleValue,
+  surveyCategory,
+}) {
+  return (
+    <div className="singleQuestion">
+      {surveyLists && (
+        <SurveyBox>
+          {surveyLists.answer.map((answers, id) => (
+            <ListBox key={id}>
+              <CheckBox
+                name={surveyCategory}
+                type="radio"
+                value={id + 1}
+                onChange={handleInputValue}
+                defaultchecked={
+                  singleValue.surveyCategory === id + 1 ? true : false
```

Comment:
```suggestion
                  singleValue.surveyCategory === id + 1
```
이미 === 평가식 자체가 true or false로 평가되기 때문에 굳이 삼항연산자 쓸 필요 없습니다!

Reply:
여러가지로 시도 해 봤는데 이부분이 잘 해결이 되지 않습니다.. 클릭한 후에 다음 페이지로 가면, 그 클릭했던 위치에 동일하게 체크가 되는 현상이 생깁니다. 라이브 코드 리뷰시에 봐 주시면 감사하겠습니다.